test/benchmarks/netperf/launch.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Launch the netperf tests
+#
+# 1. Launch the netperf-orch service
+# 2. Launch the worker pods
+# 3. Wait for the output csv file to show up in GCS
+
+import os
+import jinja2
+import subprocess
+import time
+import datetime
+from optparse import OptionParser
+
+CLOUD_STORAGE_FOLDER = "gs://gkalele-netperf-archive/"
+DEBUG_LOG = "output.txt"
+NAMESPACE = "netperf"
+
+# Moved to public Docker hub
+NETPERF_IMAGE        = "girishkalele/netperf-latest"
+
+# Unique run id for storing datafiles
+RUN_UUID = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M")
+
+# Load jinja template for the netperf workers
+POD_TEMPLATE = "netperf-rc.yaml.in"
+
+parser = OptionParser()
+parser.add_option("-n", "--iterations", dest="iterations", metavar="NUMBER", type="int", default=1,
+                  help="Number of iterations to run")
+parser.add_option("-H", "--hostnetworking", action="store_true", dest="hostnetworking", default=False,
+                  help="(boolean) Enable Host Networking Mode for PODs")
+parser.add_option("-t", "--tag", dest="tag", metavar="STRING", default=RUN_UUID,
+                  help="Tag to use for this run and storage")
+# This script has the ability to automatically push results to GCS storage buckets.
+parser.add_option("-g", "--gcsstorage", action="store_true", dest="gcsstorage", default=False,
+                  help="(boolean) Enable GCS storage for generated data and image files")
+parser.add_option("-i", "--image", dest="netperf_image",
+                  metavar="NETPERF_CONTAINER_IMAGE_PATH",
+                  default=NETPERF_IMAGE,
+                  help="Netperf container image to use for this run")
+
+def cmd(cmdline, quiet=True):
+  if quiet:
+      cmdline = cmdline + ">/dev/null 2>&1"
+  return os.system(cmdline)
+
+def load_template():
+  traw = open(POD_TEMPLATE).read()
+  t = jinja2.Template(traw)
```

Comment:
convert file to json like https://github.com/kubernetes/kubernetes/blob/master/test/e2e/serviceloadbalancers.go#L268